src/App.jsx

```diff
@@ -13,7 +15,7 @@ class App extends React.Component {
   render() {
     return (
       <div className="main">
-
+        <RecipeTile />
```

Comment:
This is just my opinion but I think it's better to not have anything here yet, because when we merge this, it's hard to test our each component on the live-server with other components together.

Reply:
yes I agree with you, that was my bad I was testing and I left it on there when I pushed